content/docs/extensions/model/sklearn.md

```diff
@@ -0,0 +1,69 @@
+# Scikit-Learn Models Support
```

Comment:
The URLs and titles aren't very consistent: `sklearn` vs. **Scikit-Learn Models Support**. Consistency reduces confusion and can help SEO.

Reply:
Up to you on this. Just keep in mind URL vs page titles please 🙂 If they're different it should be a reasoned, conscious decision, I think.